docs/Reply.md

```diff
@@ -94,6 +94,7 @@ fastify.get('/async-await', options, async function (request, reply) {
   return res
 })
 ```
+If you want to know more check out [here](https://github.com/fastify/fastify/blob/master/docs/Routes.md#async-await)!
```

Comment:
"here" links are terrible form. How about:

`If you want to know more, please review [Routes#async-await](...).`